Sources/Build/BuildDescription/SwiftTargetBuildDescription.swift

```diff
@@ -671,21 +673,41 @@ public final class SwiftTargetBuildDescription {
         let path = self.tempsPath.appending("output-file-map.json")
         let stream = BufferedOutputByteStream()
 
-        stream <<< "{\n"
-
         let masterDepsPath = self.tempsPath.appending("master.swiftdeps")
-        stream <<< "  \"\": {\n"
+        stream.send(
+            #"""
+            {
+              "": {
+
+            """#
+        )
         if self.buildParameters.useWholeModuleOptimization {
             let moduleName = self.target.c99name
-            stream <<< "    \"dependencies\": \"" <<< self.tempsPath.appending(component: moduleName + ".d")
-                .nativePathString(escaped: true) <<< "\",\n"
+            stream.send(
+                #"""
+                    "dependencies": "\#(
+                    self.tempsPath.appending(component: moduleName + ".d").nativePathString(escaped: true)
+                )",
+
+                """#
+            )
             // FIXME: Need to record this deps file for processing it later.
-            stream <<< "    \"object\": \"" <<< self.tempsPath.appending(component: moduleName + ".o")
-                .nativePathString(escaped: true) <<< "\",\n"
+            stream.send(
+                #"""
+                    "object": "\#(
+                    self.tempsPath.appending(component: moduleName + ".o").nativePathString(escaped: true)
+                )",
+
+                """#
+            )
         }
-        stream <<< "    \"swift-dependencies\": \"" <<< masterDepsPath.nativePathString(escaped: true) <<< "\"\n"
+        stream.send(
+            #"""
+                "swift-dependencies": "\#(masterDepsPath.nativePathString(escaped: true))"
+              },
```

Comment:
some lines in the string output here used 2-space indentation inconsistently, preserving just in case